### DISCUSSION TITLE
Connecting minio storage from node with private bucket

### DISCUSSION THREAD

**Discussion by nikhil104iic** [minio/minio-js#1330] [Q&A] [answered]
We are using minio storage for storing data in my project. I have to access minio from Node server, below is the configuration I am using -

`export default new MinioClient({
  endPoint: config.minioEndPoint,
  port: 9000,
  useSSL: false,
  accessKey: config.minioAccessKey,
  secretKey: config.minioSecretKey,
});`

It works fine when my bucket settings is public (Access policy). I want to access my bucket with access policy as private.

![image](https://github.com/user-attachments/assets/48ae97a4-7762-4d3e-a649-1592af67495a)

I tried useSSL: true, above but it did not work. I also tried port: 443. It did not work either. Does anyone have any idea how to connect to private buckets. ?

**Comment by prakashsvmx** [comment, marked as answer]
Bucket policy ( Anonymous Policy)  is different from IAM policy, which grants access to a user with policy actions. 
https://min.io/docs/minio/linux/administration/identity-access-management/policy-based-access-control.html#minio-policy
